test/const.go

```diff
@@ -22,7 +22,7 @@ const (
 	// for positive proof that the thing didn't happen. A negative timeout result
 	// never proves that the thing won't happen in 1 more millisecond.
 	//
-	DefaultFailureTimeout = 1 * time.Second
+	DefaultFailureTimeout = 30 * time.Minute
```

Comment:
Its safe, but it will make the tests slower. I was afraid that 1s would give us false passes. That timeout isn't used much, and it really isn't a good idea - I'd like to come up with a better solution to "negative" tests (i.e. I don't expect any output here)

Reply:
Oh crap! I just saw it -  that's supposed to be 30 *seconds*. Will fix!!!